features/install-cli.feature

```diff
@@ -52,6 +54,15 @@ Feature: Command behavior of client side tools installation
     Then the exit status should be 0
     And the binary "docker" should be installed
 
+    When I run `bundle exec vagrant service-manager install-cli docker --cli-version 1.12.1`
+    Then the exit status should be 0
+    And the binary "docker" should be installed with version "1.12.1"
+
+    # '../vagrant.d' maps to ENV['VAGRANT_HOME']
+    When I run `bundle exec vagrant service-manager install-cli docker --path ../vagrant.d/docker`
```

Comment:
well, usually backtick operator expands variables, right? Is this not the case here?